maint/gen_binding_c.py

```diff
@@ -62,6 +62,7 @@ def main():
     dump_Makefile_mk(binding_dir + "/Makefile.mk")
     dump_mpir_impl_h("src/include/mpir_impl.h")
     dump_errnames_txt(binding_dir + "/errnames.txt")
+    dump_mpi_proto_h("src/include/mpi_proto.h")
```

Comment:
this is great.

Reply:
Do we need ship a single static `mpi.h`? if that is desired, then we still need copy the content to mpi.h in autogen or configure